packages/kbn-openapi-generator/src/template_service/templates/api_client_supertest.handlebars

```diff
@@ -29,9 +31,9 @@ export function SecuritySolutionApiProvider({ getService }: FtrProviderContext)
       * {{{description}}}
       */
     {{/if}}
-    {{camelCase operationId}}({{#if (or requestQuery requestParams requestBody)}}props: {{operationId}}Props{{/if}}) {
+    {{camelCase operationId}}({{#if (or requestQuery requestParams requestBody)}}props: {{operationId}}Props, {{/if}}kibanaSpace: string = 'default') {
```

Comment:
nit: TS is able to infer string type from the literal. It's enough to define `kibanaSpace = 'default'`.